spec/scheduling-tasks.md

```diff
@@ -272,49 +362,105 @@ Issue: [=Run steps after a timeout=] doesn't necessarily account for suspension;
 [whatwg/html#5925](https://github.com/whatwg/html/issues/5925).
 
 <div algorithm>
-  To <dfn>select the scheduler task queue</dfn> for a {{Scheduler}} |scheduler| given an
-  {{AbortSignal}} or null |signal|, and a {{TaskPriority}} or null |priority|:
-
-  1. If |priority| is null, |signal| is not null and [=implements=] the {{TaskSignal}} interface,
-     and |signal| [=TaskSignal/has fixed priority=], then set |priority| to |signal|'s
-     [=TaskSignal/priority=].
-  1. If |priority| is null and |signal| is not null and [=implements=] the {{TaskSignal}} interface,
-     then
+  To <dfn>schedule a yield continuation</dfn> for {{Scheduler}} |scheduler| given
+  {{SchedulerYieldOptions}} |options|:
+
+  1. Let |result| be [=a new promise=].
+  1. Let |abortOption| be |options|["{{SchedulerYieldOptions/signal}}"] if it [=map/exists=],
```

Comment:
Note: Should this be called abortOption given it could be used for more things?

Why not just `signalOption`, it would be less confusing even if it only supported abortcontroller.

Reply:
Yeah good point. It's either change to signalOption or let priorityOption also be a signal. Changed to signalOption.